.github/workflows/site.yml

```diff
@@ -0,0 +1,39 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to you under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: Build Website and Javadoc
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled]
+    paths:
+      - 'site/**'
+    branches:
+      - '*'
+
+jobs:
+  build-website:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Build site
+        working-directory: site
+        run: |
+          docker compose run -e JEKYLL_UID=$(id -u) -e JEKYLL_GID=$(id -g) build-site
```

Comment:
In the latest iteration of the site publishing workflows, we no longer need to set `JEKYLL_UID` and `JEKYLL_GID`, so I those 2 environment variables can be removed.

See: https://github.com/apache/calcite/blob/main/.github/workflows/publish-non-release-website-updates.yml#L59

Reply:
Thanks for the review Francis! Fixed in https://github.com/apache/calcite/pull/4173/commits/6b04e32bff588108184e5475987c394e5924fe0c